partner_firstname/models.py

```diff
@@ -38,12 +38,16 @@ class ResPartner(models.Model):
         required=False,
         store=True)
 
+    @api.model
+    def _get_computed_name(self, lastname, firstname):
+        """Write the 'name' field according to splitted data."""
+        return u" ".join((p for p in (lastname, firstname) if p))
```

Comment:
I think we need more feedback before changing the default behaviour. Now this PR is here to improve https://github.com/OCA/hr/pull/128. 

Would you agree to leave this PR as is and have another PR or issue to change the default behaviour and gather feedback on that specific point.